app/pulp/app/models/progress.py

```diff
@@ -58,6 +62,9 @@ class ProgressReport(Model):
     task = models.ForeignKey("Task", on_delete=models.CASCADE)
     suffix = models.TextField(default='')
 
+    _using_context_manager = False
```

Comment:
Is the database going to reject these attributes? Have you tried calling Save() within and outside of a context manager with this PR? I can't test it because I am working on Pulp2 stuff currently. I'm not sure if Django will ignore these attributes or not do you know?